src/services/video-conference-manager/types.ts

```diff
@@ -44,6 +45,11 @@ export interface Locale {
   };
 }
 
+export interface Avatar {
+  thumb: string;
+  link: string;
+}
+
```

Comment:
You don't need to create this interface here, there is already an `Avatar` interface that you can use

Reply:
https://github.com/SuperViz/sdk/blob/6e5e6ae210389de959057dd32b235215d89fe16b/src/common/types/user.types.ts#L16